scripts/combine/setupCombine.py

```diff
@@ -1072,7 +1060,7 @@ def fake_nonclosure(h, axesToDecorrNames, param_idx=1, variation_size=0.5, norma
     cardTool.addSystematic("muonScaleSyst_responseWeights",
         processes=['single_v_samples'],
         group="scaleCrctn",
-        splitGroup={f"muonCalibration" : f".*"},
+        splitGroup={f"muonCalibration" : f".*", "experiment": ".*"},
```

Comment:
historically it was intentional that the calibration uncertainties were not included in "experiment" since this grouping was intended as "experiment except for calibration" for the purposes of the breakdown of uncertainties for mZ with the dilepton mass.

We probably want to keep the ability to do that in case we need to run the dilepton mass mZ again (in principle we should to be fully up to date, since there have been some small changes to theory, efficiencies etc since we originally produced those results)

For w-like and mW I agree it makes sense to have a group which really contains all of the experimental uncertainties including the calibration ones.

Reply:
OK I added another group "expNoCalib" for this